backend/main.go

```diff
@@ -53,6 +53,11 @@ func main() {
 		port = "5000"
 	}
 
+	apiKey, ok := os.LookupEnv("API_KEY")
+	if !ok {
+		log.Fatal().Msg("API_KEY is required")
```

Comment:
Shouldn't be Fatal though. Should be a warning that the `API_KEY` is not set, therefore you can't make API calls to endpoints that requires an API key.

Reply:
[5f24f28](https://github.com/teknologi-umum/semyi/pull/39/commits/5f24f28188dce5593279e5cc4a9ed957a05f2d37)